### ISSUE TITLE
Where to take pbounds from in order to perform optimization?

### ISSUE THREAD

**Issue by AndrewLaganaro** [fmfn/BayesianOptimization#330]
Hello, my question is how to discover the **pbounds values** in order to **perform optimization**?

On the example code:

```
# Bounded region of parameter space
pbounds = {'x': (2, 4), 'y': (-3, 3)}
```
I'm new to his method, so sorry if the question is way too obvious for some.

As far as I can understand, 
Lets say that I have a dataset with a **target variable**, say, "**sales**" that I want to predict with **linear regression** with sklearn.

Would it be the right choice to take **sales max and min value** to use it as pbounds?

Or pbounds refer to some of the **error metrics** usually found in models like **MSE, RMSE** and etc that we take after the first training?

As far as I can understand, my code would look like this after importing, but I lack the instruction on how to get pbounds from
```
from sklearn.linear_model  import LinearRegression
from bayes_opt import BayesianOptimization

# model
lr_model = LinearRegression().fit(x_train, y_train)
# x_train and y_train already exists from previous lines...
 
# prediction
# yhat_lr = lr_model.predict(x_test)

# Bounded region of parameter space
pbounds = {'x': (2, 4), 'y': (-3, 3)}  # take pbounds from some variable(?????)

optimizer = BayesianOptimization(f=lr_model, pbounds=pbounds, random_state=1)

optimizer.maximize(init_points=2, n_iter=3)

print(optimizer.max)
#etc...
```

Hope I made my question clear! It's my first time testing this library and I really don't know where to find a variable to take the pbounds from. Really thanks!

**Comment by bwheelz36** [comment]
Hey @AndrewLaganaro. If I understand you correctly, fundamentally your goal here is to **predict** some metric (sales). 

This repository is not primarily concerned with regression (prediction), although it makes use of regression to perform optimisation. Instead you could take a look through [sklearn](https://scikit-learn.org/stable/modules/gaussian_process.html). I've linked you to gaussian process regression because that is what we use in this repo, but there are about a million and one ways to perform regression - in any case sklean is a good place to start.
